pyproject.toml

```diff
@@ -39,12 +39,14 @@ structlog = {version = ">=20.0.0", optional = true}
 aiohttp = {version = ">=3.7.0", optional = true}
 cryptography = ">=3.3.1"
 
+# TODO: We should migrate from asynctest to pytest-asyncio when we raise our minimum Python version to 3.7
```

Comment:
Should we create an issue to raise the minimum Python version to 3.7 with this item in the acceptance criteria? We can also leave this comment here.

Reply:
Done: https://github.com/HathorNetwork/python-hathorlib/issues/15